drivers/dma/dma_andes_atcdmac300.h

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2023 Andes Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_DMA_ANDES_ATCDMAC300_H_
```

Comment:
Are there going to be other variants of this dma controller implemented? If not then the header file contents could be placed in the .c file directly.

Reply:
No, there will only be this implemented.
We have update the source code to integrate them.

Thanks a lot.